src/test/java/org/aksw/gerbil/execute/KETest.java

```diff
@@ -0,0 +1,145 @@
+										new TypedNamedEntity(0, 10, "http://aksw.org/notInWiki/Brad_Smith", new HashSet<String>(Arrays.asList(
+						            "http://www.w3.org/2002/07/owl#Individual",
+						            "http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Person")))), 
+								new RelationImpl(new TypedNamedEntity(0, 10, "http://aksw.org/notInWiki/Brad_Smith", new HashSet<String>(Arrays.asList(
+								            "http://www.w3.org/2002/07/owl#Individual",
+								            "http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Person"))),
+												new Annotation("http://dbpedia.org/ontology/employer"),
+												new TypedNamedEntity(14, 23, "http://dbpedia.org/resource/Microsoft", new HashSet<String>(Arrays.asList(
+								            "http://www.w3.org/2002/07/owl#Individual",
+								            "http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Organisation"))))))},
+				GOLD_STD, Matching.STRONG_ANNOTATION_MATCH, new double[] { 0.6666666666666666, 0.6666666666666666, 0.6666666666666666, 0.6666666666666666, 0.6666666666666666, 0.6666666666666666, 0 } });
+	testConfigs.add(new Object[] {
+				new Document[] { new DocumentImpl(TEXTS[0], "doc-0",
+						Arrays.asList((Marking) new RelationImpl(
+								new TypedNamedEntity(14, 23, "http://dbpedia.org/resource/Microsoft", new HashSet<String>(Arrays.asList(
+						            "http://www.w3.org/2002/07/owl#Individual",
+						            "http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Organisation"))),
+								new Annotation("http://dbpedia.org/ontology/president"),
+								new TypedNamedEntity(0, 10, "http://aksw.org/notInWiki/Brad_Smith", new HashSet<String>(Arrays.asList(
+						            "http://www.w3.org/2002/07/owl#Individual",
+						            "http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Person")))))) },
+				GOLD_STD, Matching.STRONG_ANNOTATION_MATCH, new double[] { 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0 } });
+	return testConfigs;
+}	
+	private Document annotatorResults[];
+	private Document goldStandards[];
+	private double expectedResults[];
+	private Matching matching;
+
+	public KETest(Document[] annotatorResults, Document[] goldStandards, Matching matching,
+			double[] expectedResults) {
+		this.annotatorResults = annotatorResults;
+		this.goldStandards = goldStandards;
+		this.expectedResults = expectedResults;
+		this.matching = matching;
+	}
+	
+	@Test
+	public void test() {
+		int experimentTaskId = 1;
+		SimpleLoggingResultStoringDAO4Debugging experimentDAO = new SimpleLoggingResultStoringDAO4Debugging();
+		ExperimentTaskConfiguration configuration = new ExperimentTaskConfiguration(
+				new TestAnnotatorConfiguration(Arrays.asList(annotatorResults), ExperimentType.KE),
+				new TestDataset(Arrays.asList(goldStandards), ExperimentType.KE), ExperimentType.KE, matching);
+		runTest(experimentTaskId, experimentDAO, new HTTPBasedSameAsRetriever(), new EvaluatorFactory(URI_KB_CLASSIFIER), configuration,
```

Comment:
Using a `new HTTPBasedSameAsRetriever(),` here is not a very good idea. You won't find it in the other tests of this package since it a) creates a lot of additional effort and b) will make the test result dependent form an internet connection. Please remove it.